.github/workflows/angular.yml

```diff
@@ -4,26 +4,25 @@
 name: Angular CI
 
 on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
+    push:
+        branches: [main]
+    pull_request:
+        branches: [main]
 
 jobs:
-  build:
+    build:
+        runs-on: ubuntu-latest
 
-    runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                node-version: [12, 14]
```

Comment:
Feel free to drop 12 (as this build fails with 12) as I've removed 12 as a required pass -- you "should" be able to add 16 as well -- if not I can add later as part of preparing for release.